operators/project.py

```diff
@@ -351,12 +351,15 @@ def on_done(future):
             context.scene.dream_textures_info = ""
             context.scene.dream_textures_progress = 0
             generated = future.result()
+            gen_seed_string = str(generated.seeds[0])
+            prompt_subject = context.scene.dream_textures_project_prompt.prompt_structure_token_subject
+            name_with_prompt = str(prompt_subject + " " + gen_seed_string)
```

Comment:
Can we use an f-string here too?

> I included the first option of the above comment in this suggestion, but if you think the seed on the end is better that would work too

```suggestion
            prompt_subject = context.scene.dream_textures_project_prompt.prompt_structure_token_subject
            name_with_prompt = f"{generated.seeds[0]} ({prompt_subject})"
```